schema/SasImplementationMessage.schema.json

```diff
@@ -1,47 +1,48 @@
 {
   "$schema": "http://json-schema.org/draft-04/schema#",
-    "description": "SAS-SAS Implementation Record Exchange",
-    "type": "object",
-    "required": [
-      "id",
-      "name",
-      "administratorId",
-      "contactInformation",
-      "publicKey",
-      "fccInformation",
-      "url"
-    ],
-    "additionalProperties":false,
-    "properties": {
-      "id": {
-        "type": "string",
-        "description": "Unique designator for the Sas Implementation Record."
-      },
-      "name": {
-        "type": "string",
-        "description": "The name for the Sas Implementation Record."
-      },
-      "administratorId": {
-        "type": "string",
-        "description": "Reference Id of Sas Administrator Object"
-      },
-      "contactInformation": {
-        "$ref": "file:ContactInformation.schema.json"
-      },
-      "publicKey": {
-        "type": "string",
-        "description": "Public Key in format X.509"
-      },
-      "fccInformation": {
-        "$ref": "file:FccInformation.schema.json",
-        "description": "Contains the FCC certification information for this SAS implementation."
-      },
-      "url": {
-        "type": "string",
-        "description": "Publicly addressable URL for the SAS"
-      }
+  "description": "SAS-SAS Implementation Record Exchange",
+  "type": "object",
+  "required": [
+    "id",
+    "name",
+    "administratorId",
+    "contactInformation",
+    "publicKey",
+    "fccInformation",
+    "url"
+  ],
+  "additionalProperties": false,
+  "properties": {
+    "id": {
+      "type": "string",
+      "description": "Unique designator for the Sas Implementation Record.",
+      "pattern": "((.+?)\/(.+?)\/(.+?)+)"
+    },
+    "name": {
+      "type": "string",
+      "description": "The name for the Sas Implementation Record."
+    },
+    "administratorId": {
+      "type": "string",
+      "description": "Reference Id of Sas Administrator Object"
+    },
+    "contactInformation": {
+      "$ref": "file:ContactInformation.schema.json"
+    },
+    "publicKey": {
+      "type": "string",
+      "description": "Public Key in format X.509"
+    },
+    "fccInformation": {
+      "$ref": "file:FccInformation.schema.json",
+      "description": "Contains the FCC certification information for this SAS implementation."
+    },
+    "url": {
+      "type": "string",
+      "description": "Publicly addressable URL for the SAS",
+      "pattern": "(?:https?)://[-a-zA-Z0-9.]+(:(6553[0-5]|655[0-2][0-9]|65[0-4][0-9][0-9]|6[0-4][0-9][0-9][0-9]|\\d{2,4}|[1-9]))?"
```

Comment:
See https://spacetelescope.github.io/understanding-json-schema/reference/string.html#format

I think you can use the format types for good string validation od the dates, emails, urls we use.